src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -67,12 +53,15 @@ def _post(self, request: Request):
         version = request.GET.get("version") or "1"
         set_tag("relay_protocol_version", version)
 
-        if self._should_use_v3(version, request):
+        if version == "4" and request.relay_request_data.get("global"):
+            response["global"] = get_global_config()
```

Comment:
As mentioned by others, let's make sure that this is in sync with the Relay implementation.